_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -580,3 +595,16 @@ def get_geo_values(self, geo_type):
             crosswalk = pd.read_csv(stream, dtype=str)
             self.geo_lists[geo_type] = set(crosswalk[geo_type])
             return self.geo_lists[geo_type]
+
+    def _megafips_to_zero(self, fips_list):
+        output = set()
+        fips_set = set(fips_list)
+        state_fips_codes = {str(x).zfill(2) + "000" for x in range(1, 73)}
```

Comment:
should we be using the actual list of state codes here instead of `range(1,73)`?

Reply:
yea good idea, this was just the jhu copy paste. thanks for calling out, will fix